tests/utils.py

```diff
@@ -47,3 +47,13 @@ def wrapper(cls):
         return dataclasses.dataclass(cls)
 
     return wrapper
+
+
+def get_field(cls, field_name):
+    matches = [
+        field for field in cls._type_definition.fields if field.name == field_name
+    ]
+    try:
+        return matches[0]
+    except IndexError:
+        raise ValueError(f"{cls} does cont contain a field named {field_name}")
```

Comment:
Is this required? I feel like we could use [TypeDefinition.get_field](https://github.com/strawberry-graphql/strawberry/blob/main/strawberry/types/types.py#L108) from strawberry directly

Reply:
TIL, thank you!